carAPI/src/test/java/boundary/CarResourceTests.java

```diff
@@ -0,0 +1,88 @@
+package boundary;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import com.everis.boundary.CarResource;
+import com.everis.control.CarService;
+import com.everis.entity.Car;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CarResourceTests {
+
+	@InjectMocks
+	private CarResource carResource;
+	
+	@Mock
+	private CarService carService;
+	
+	@Mock
+	private UriInfo uriInfo;
+	
+	private Car car;
```

Comment:
you can make this variable final, this way you cannot change the car properties n any test having side effects for other tests. Final variables (a.k.a constants) are usually a good practice in programming.